src/pages/settings/bank-accounts/components/index.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+export { default as Details } from './Details';
+export { default as ConnectAccounts } from './ConnectAccounts';
```

Comment:
No default exports, so you most likely won't need this file.

Reply:
> No default exports, so you most likely won't need this file.

Removed